File: get_batch.py
import re
import os

with open('lore.html', 'r', encoding='utf-8') as f:
    html = f.read()

start = html.find('const loreEntries = [')
end = html.find('];', start)
entries_str = html[start+20:end+1]

# Each entry is enclosed in {}
entries = re.findall(r'\\{([^}]+)\\},', entries_str)

batch = []
for entry in entries:
    # check category
    if \"'category': '場所'\" in entry or '\"category\": \"場所\"' in entry:
        # get url
        url_match = re.search(r\"'url':\\s*'([^']+)'\", entry)
        if url_match:
            url = url_match.group(1)
            # check draft
            if os.path.exists(url):
                with open(url, 'r', encoding='utf-8') as f:
                    content = f.read()
                    if '※このページは作成中のドラフト記事です' in content:
                        batch.append(url)
    
    if len(batch) >= 10:
        break

print(\"\\n\".join(batch))

File: get_batch.js
const fs = require('fs');
const html = fs.readFileSync('lore.html', 'utf8');
const start = html.indexOf('const loreEntries = [');
const entries = eval(html.substring(start + 20, html.indexOf('];', start) + 1));

let placeholders = [];
for (let e of entries) {
    if (e.category === '場所') {
        if (fs.existsSync(e.url)) {
            let content = fs.readFileSync(e.url, 'utf8');
            if (content.includes('このページは作成中のドラフト記事です')) {
                placeholders.push(e.url);
            }
        }
    }
}
console.log(`Found ${placeholders.length} placeholder places.`);
console.log(placeholders.slice(0, 10).join('\\n'));
